scraper/scraper.js

```diff
@@ -54,7 +54,7 @@ class Scraper {
       const img = $card.find(selectors.img).html().match(regEx.img)[0];
       const $title = $card.find(selectors.name);
       const name = $title.text().trim();
-      const link = `${baseUrl}${$title.attr('href')}`;
+      const link = `${baseUrl}${$title.attr('href').slice(1}`;
```

Comment:
Hey, thanks for finding and fixing this. I think you are just missing a closing `)` after the 1 and then we can merge this MR. Can you please fix that?